zallet/src/components/json_rpc/methods/list_unspent.rs

```diff
@@ -127,6 +175,66 @@ pub(crate) fn call(wallet: &DbConnection) -> Response {
             .key_derivation()
             .map(|derivation| u32::from(derivation.account_index()));
 
+        let utxos = wallet
```

Comment:
How can we test that this logic is what we expect it to be ? Is there some manual testing you did when creating this pull request that can be shared ?